test/viz/index.spec.ts

```diff
@@ -19,8 +23,41 @@ describe('End-to-end-tests for viz', () => {
     jest.clearAllMocks();
   });
 
-  const hasFile = async (...paths: string[]): Promise<boolean> => {
-    return fs.exists(path.resolve(constants.PWD, ...paths));
+  test('happy path', async () => {
+    const vizName = vizNames.happyPath;
+    const config: VizConfig = {
+      devBucket: 'test/dscc-gen-test-dev',
+      prodBucket: 'test/dscc-gen-test-prod',
+      yarn: false,
+      codelab: false,
+      projectName: vizName,
+      projectChoice: ProjectChoice.VIZ,
+      basePath: '.',
+    };
+
+    await sut.createFromTemplate(config);
+    expect(await hasFile(vizName)).toBeTruthy();
+    expect(await hasFile(vizName, 'src', 'index.js')).toBeTruthy();
+
+    await files.remove(vizName);
+  });
+});
+
+describe('End-to-end-tests for viz codelab', () => {
```

Comment:
Very minor nit, but this should probably be in the `end-to-end-tests for viz describe block, not its own describe block.